container-storage-setup.sh

```diff
@@ -888,6 +876,38 @@ create_disk_partitions() {
   done
 }
 
+remove_partition() {
+  local dev="$1"
+
+  if [ -x "/usr/sbin/parted" ]; then
+    parted "$dev" rm 1
+  else
+    sfdisk --delete "$dev" 1
+  fi
+}
+
+# Remove disk pvs and partitions. This is called in reset storage path.
+# If partition or pv does not exist, it will still return success. Error
+# willl be returned only if pv or partition existst and removal fails.
+remove_disk_pvs_parts() {
+  local devs="$1" part
+
+  for dev in $devs; do
+    part=$(dev_query_first_child $dev)
+    [ -z "$part" ] && continue
+
+    if ! remove_pv_if_exists $part; then
+      Error "Failed to remove physical volume label on device $part"
+      return 1
```

Comment:
And we can change it later if there is a good use case. Right now model is that on first failure bail out. We probably should implement a switch say --force. That switch should do whatever cleanup is possible. 

`dmsetup remove -f foo`

And with that switch it will make sense to continue to remove pvs from other disks.